drakrun/drakrun/vm.py

```diff
@@ -74,13 +74,14 @@ def delete_vm_conf(vm_id: int) -> bool:
 
 
 class VirtualMachine:
-    def __init__(self, backend: StorageBackendBase, vm_id: int) -> None:
+    def __init__(self, backend: StorageBackendBase, vm_id: int, fmt: str = None) -> None:
         self.backend = backend
         self.vm_id = vm_id
+        self._format = "vm-{}" if fmt is None else fmt
```

Comment:
I think that we could remove this line whole line by setting `fmt: str = "vm-{}"` in the function signature

Reply:
ah, missed out this during multiple refactors